includes/internal/emails/views/html-settings.phtml

```diff
@@ -0,0 +1,23 @@
+<form id="email-notification-settings-form" action="options.php" method="post">
+	<?php echo settings_fields( 'sensei-settings' ); ?>
+	<?php
+	foreach ($options as $option_name => $option_data) {
+		$this->form_field_hidden( $option_name, $option_data );
```

Comment:
We have to output hidden fields with all the options here.
Otherwise it will complain on required ones.